ui/src/shared/components/DeleteDataForm/BucketsDropdown.tsx

```diff
@@ -22,7 +22,22 @@ const BucketsDropdown: FunctionComponent<Props> = ({
   bucketName,
   onSetBucketName,
 }) => {
-  const bucketNames = buckets.map(bucket => bucket.name)
+  const handleDefaults = name => name === '_monitoring' || name === '_tasks'
+  const bucketNames = buckets
+    .map(bucket => bucket.name)
+    .sort((a, b) => {
```

Comment:
what do you think about moving this line out of the component and into the model (i.e. a reducer or maybe selector)? this seems to be an issue that crops up a bit, and it seems like we want buckets to be sorted properly. Doing this in reduxland will ensure future uses of buckets are already sorted out of the box, and all sorting is consistent (even if it's not quite right).

Reply:
I think that sounds like a great idea. Lezdoit